Tests/TuistKitTests/ProjectEditor/ProjectEditorMapperTests.swift

```diff
@@ -35,15 +37,19 @@ final class ProjectEditorMapperTests: TuistUnitTestCase {
         let (project, graph) = try subject.map(tuistPath: tuistPath,
                                                sourceRootPath: sourceRootPath,
                                                xcodeProjPath: xcodeProjPath,
+                                               setupPath: setupPath,
+                                               configPath: configPath,
                                                manifests: manifestPaths,
                                                helpers: helperPaths,
                                                templates: templates,
                                                projectDescriptionPath: projectDescriptionPath)
 
         // Then
         let targetNodes = graph.targets.values.lazy.flatMap { targets in targets.compactMap { $0 } }.sorted(by: { $0.target.name < $1.target.name })
-        XCTAssertEqual(targetNodes.count, 3)
-        XCTAssertEqual(targetNodes.last?.dependencies, Array(targetNodes.dropLast()))
+
+        // expecting `targetNodes == [Config, ProjectDescriptionHelpers, Setup, Templates, TemporaryDirectory.XXXMainifest]`
+        XCTAssertEqual(targetNodes.count, 5)
+        XCTAssertEqual(targetNodes.last?.dependencies.sorted(by: { $0.name < $1.name }), Array(targetNodes.dropLast()))
```

Comment:
Rather than sorting, you can wrap the arrays into `Set` which is more descriptive that you don't care about the order

Reply:
What if `targetNodes.last?.dependencies` contains duplication which I would like to catch in unit tests?